### PR TITLE
Support for aws-sdk-* 0.15

### PR DESCRIPTION
**Pull request by sciguy16**
Another week, another aws-sdk-dynamodb & streams release.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -102,11 +102,11 @@
 //!
 //! ```toml
 //! [dependencies]
-//! serde_dynamo = { version = "3", features = ["aws-sdk-dynamodb+0_12"] }
+//! serde_dynamo = { version = "3", features = ["aws-sdk-dynamodb+0_15"] }
```
Comment (bryanburgers) [inline src/lib.rs:105]: 💚

**Comment by bryanburgers** [comment]
Thank you! I’ll try to get this merged and pushed to crates.io tomorrow.

**Comment by sciguy16** [comment]
@bryanburgers It looks like I can clear the clippy errors by just deleting all of the extra `'a`s - would you like that in a separate PR?

**Comment by bryanburgers** [comment]
> @bryanburgers It looks like I can clear the clippy errors by just deleting all of the extra `'a`s - would you like that in a separate PR?

A separate PR would be preferable to this PR. Otherwise that’s a chore I can deal with tomorrow. No big deal.

**Comment by bryanburgers** [comment]
Released this morning! Thanks for your help!

**Pull request merged**